Zend/zend_vm_def.h

```diff
@@ -2111,12 +2111,14 @@ ZEND_VM_C_LABEL(fetch_obj_r_fast_copy):
 				}
 			}
 			name = Z_STR_P(GET_OP2_ZVAL_PTR(BP_VAR_R));
+			zend_string_hash_val(name);
```

Comment:
I don't think this one is necessary, all `CONST` string operands should have the hash calculated.